src/release-info/intro.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<preambles>
+  <eac>
+  <!--
+ 
+                              An XML Schema:
+  Encoded Archival Context - Corporate Bodies, Persons, and Families (EAC-CPF)
+
+  See: https://eac.staatsbibliothek-berlin.de/schemata-and-tag-library/
+
+  *****************************************************************
+  *                                                               *                
+  *                            EAC-CPF 2.0                        *
+  *                                                               *
+  *                           2022 July 17                        *          
+  *                                                               *
+  *****************************************************************
+
+  *****************************************************************
+  
+  EDITORS: Society of American Archivists Technical Subcommittee on Encoded Archival Standards (TS-EAS)
+  
+  PUBLISHER: Society of American Archivists in collaboration with the Staatsbibliothek zu Berlin
+  
+  PURPOSE: The EAC-CPF Schema is a communication standard for the description of corporate bodies,
+  persons, and families. It is based on the International Council of Archives' International Standard 
+  Archival Authority Record For Corporate Bodies, Persons and Families (ISAAR(CPF)). For additional
+  documentation on the use of the standard, see the EAC-CPF Tag Library
+  (http://eac.staatsbibliothek-berlin.de/schemata-and-tag-library/).
+   
+  *****************************************************************
+ 
+  This schema has been generated automatically. 
+   
+  If you would like to contribute to this project, please see:
+  https://github.com/SAA-SDT/eac-cpf-schema/blob/development/CONTRIBUTING.md
+  
+  Comments, questions, and suggestions may be addressed to: 
+  ts-eas@archivists.org
+  
+  For more information about the EAC-CPF team, see https://github.com/SAA-SDT/TS-EAS-subteam-notes/wiki/EAC-CPF-Sub-Team  
```

Comment:
Should be: https://github.com/SAA-SDT/TS-EAS-subteam-notes/wiki/EAC-CPF-Sub-Team

Reply:
That's what was there previously, right?

Reply:
In any event, I've added the link to a new line for now.